src/devcards/core.clj

```diff
@@ -178,13 +179,18 @@
   (when (utils/devcards-active?)
     `(devcards.util.utils/pprint-code ~obj)))
 
-(defmacro mkdn-code [body] `(str "\n```\n" ~body "```\n"))
+(defmacro mkdn-code [body] `(str "\n```clojure\n" ~body "\n```\n"))
 
 (defmacro mkdn-pprint-code [obj]
   (when (utils/devcards-active?)
     `(mkdn-code
       (devcards.util.utils/pprint-code ~obj))))
 
+(defmacro mkdn-pprint-source [obj]
+  (when (utils/devcards-active?)
+    `(mkdn-code
+       ~(or (cljs.repl/source-fn &env obj) (str "Source not found")))))
+
```

Comment:
There should be a difference between `mkdn-source` and `source`.  Where one calls mkdn code. and the other just returns the source I guess.  I don't think the `pprint` is needed in the name because pprint isn't called.